Flask_app/views/user.py

```diff
@@ -1,5 +1,5 @@
 from flask import jsonify,Blueprint
-admin = Blueprint('admin', __name__, url_prefix='/admin')
-@admin.route('/')
+bp = Blueprint('bp', __name__, url_prefix='/users')
```

Comment:
```suggestion
bp = Blueprint('bp', __name__, url_prefix='/users')
```

user_bp = Blueprint('user_bp', __name__, url_prefix='/users')